config/initializers/stripe.rb

```diff
@@ -1 +1,17 @@
 Stripe.api_key = ENV['STRIPE_SECRET_KEY']
+
+StripeEvent.configure do
+  subscribe 'charge.succeeded' do |event|
+    user = User.find_by(customer_token: event.data.object.customer)
+    Payment.create(
```

Comment:
You should use `create!` here. That way it won't fail silently, it will blow up and your error system will notify you.